pkg/orderers/internal/common/pbftslot.go

```diff
@@ -152,6 +152,10 @@ func (slot *PbftSlot) CheckCommitted() bool {
 }
 
 func (slot *PbftSlot) GetPreprepare(digest []byte) *pbftpbtypes.Preprepare {
+	if digest == nil || slot.Preprepare == nil {
```

Comment:
The check for `slot.Preprepare == nil` is redundant.

Reply:
Fair!